e2e/utils.ts

```diff
@@ -1,5 +1,5 @@
 import net from 'node:net';
-import { ConsoleMessage, test as basicTest } from '@playwright/test';
+import { type ConsoleMessage, test as basicTest } from '@playwright/test';
```

Comment:
I personally prefer separate lines:
```suggestion
import { test as basicTest } from '@playwright/test';
import type { ConsoleMessage } from '@playwright/test';
```